src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -117,17 +116,4 @@ public function add_views( $views ) {
 	public function unregister_list_widget() {
 		unregister_widget( 'Tribe__Events__List_Widget' );
 	}
-
-	/**
-	 * Filters the template vars for widget-specific items.
-	 *
-	 * @since 5.3.0
-	 *
-	 * @param array<string,mixed> $template_vars The current template variables.
-	 *
-	 * @return array<string,mixed> The modified template variables.
-	 */
-	public function filter_template_vars( $template_vars ) {
-		return tribe( Widget_List::class )->disable_json_data( $template_vars );
-	}
```

Comment:
I thought we needed to remove JSON-LD for widgets.

Reply:
Hm, I didn't remove this intentionally...looking.